### ISSUE TITLE
[REST API] Update Rule not updating model Fields

### ISSUE THREAD

**Issue by oben01** [Yadoms/yadoms#715]
I did a PUT request to /automation/rule/{ruleId} with a Request Body as :
```json
{
    "id": 5,
    "name": "bvn",
    "description": "",
    "interpreter": "yPython3",
    "editor": "code",
    "model": "testNotWorking",
    "content": "test",
    "configuration": "test",
    "autoStart": true,
    "state": "Stopped",
    "errorMessage": "",
    "startDate": "20210105T124337.486824",
    "stopDate": "20210105T124347.376593"
  }
```

Am getting the Response :

```json
{
  "result": true,
  "message": "",
  "data": {
    "id": 5,
    "name": "bvn",
    "description": "",
    "interpreter": "yPython3",
    "editor": "code",
    "model": "",
    "content": "test",
    "configuration": "test",
    "autoStart": true,
    "state": "Stopped",
    "errorMessage": "",
    "startDate": "20210105T124337.486824",
    "stopDate": "20210105T124347.376593"
  }
}
```
The field model is not updated.

After a short investigation I found that the function updateRule from the file yadoms\sources\server\database\common\requesters\Rule.cpp is not updating this value (Need to be added after line 201)

Can you please add it

**Comment by jano42** [comment]
I don't remember exactly, but it's volontary.
I think the model couln't be changed

**Comment by sgallou** [comment]
Closed by f97278e0d5aed2331a2d215305245e12fa3857e8
